website/content/docs/intro/usecases/what-is-a-service-mesh.mdx

```diff
@@ -0,0 +1,109 @@
+---
+layout: docs
+page_title: What is service mesh??
+description: >-
+  Learn what a serive mesh is, it's benefits, and how it works.
+---
+
+# What is a Service Mesh?
+
+A _service mesh_ is a dedicated network layer that provides secure service-to-service communication for on-prem, cloud, or multi-cloud infrastructure.
+Service meshes are often used with a microservice architectural pattern, but can provide value in any scenario where complex networking is involved.
+
+## Benefits of a Service Mesh
+
+A _service mesh_ provides benefits for all organziations, ranging from security to improved application resiliency.
```

Comment:
```suggestion
A service mesh provides benefits for all organizations, ranging from security to improved application resiliency.
```
Is there a reason that service mesh is italicized throughout the entire document?

Reply:
We normally use italic for words that standout, such as service names. After reviewing the style guide, we only need to use italic in the [first usage of the term](https://docs.google.com/document/d/1MRvGd6tS5JkIwl_GssbyExkMJqOXKeUE00kSEtFi8m8/edit#heading=h.lhe5v722498w).